src/helper/remote_config/protocol/cached_target_file_hash.hpp

```diff
@@ -0,0 +1,24 @@
+// Unless explicitly stated otherwise all files in this repository are
+// dual-licensed under the Apache-2.0 License or BSD-3-Clause License.
+//
+// This product includes software developed at Datadog
+// (https://www.datadoghq.com/). Copyright 2021 Datadog, Inc.
+#pragma once
+
+#include <string>
+
+namespace dds::remote_config::protocol {
+
+class cached_target_files_hash {
```

Comment:
Usually structs are preferred to classes without logic and getters/setters as they don't provide any value for the most part.

Reply:
Done